src/main/resources/com/google/api/codegen/py/method_sample.snip

```diff
@@ -0,0 +1,97 @@
+@extends "../common.snip"
+
+@snippet generateMethodSampleCode(docConfig)
+  @let prompt = ">>>", \
+       ApiName = docConfig.getApiName
+    Example:
+      @join import : docConfig.getAppImports
+        {@prompt} {@import}
+      @end
+      #
+      @join param : docConfig.getParams
+        {@prompt} {@initLine(param, docConfig)};
+      @end
+      {@prompt} api = {@ApiName}()
+      #
+      @if docConfig.isUnpagedListCallableVariant
+        # FIXME: adapt for Python
+        @let resourcesField = docConfig.getResourcesFieldForUnpagedListCallable
+          while (true) {
+            {@callResultSampleCode(docConfig.getReturnType)}{@methodCallSampleCode(docConfig, apiName)};
+            for ({@elementTypeName(resourcesField)} elements : response.{@getResourcesListCall(resourcesField)}) {
+              // doThingsWith(elements);
+            }
+            String nextPageToken = response.getNextPageToken();
+            if (!Strings.isNullOrEmpty(nextPageToken)) {
+              request = request.toBuilder().setPageToken(nextPageToken).build();
+            } else {
+              break;
+            }
+          }
+        @end
+      @else
+        @if docConfig.isCallableVariant
+          # FIXME: adapt for Python
+          ListenableFuture<{@getGenericAwareReturnType(docConfig)}> future = {@methodFutureCall(docConfig, apiName)};
+          // Do something{@BREAK}
+        @end
+        @if docConfig.isPagedVariant
+          # FIXME: adapt for Python
+          for ({@docConfig.getReturnType} elements : {@methodCallSampleCode(docConfig, apiName)}) {
+            // doThingsWith(elements);
+          }
+        @else
+          {@prompt} {@callResultSampleCode(docConfig.getReturnType)}{@methodCallSampleCode(docConfig)}
+        @end
+      @end
+  @end
+@end
+
+# Optionally render the return value from the API method call
+@private callResultSampleCode(returnType)
+  @if returnType.isEmpty
+  @else
+    response = {@""}
```

Comment:
I will interject: yes, it preserves the space between the '=' and the following expression.